modules/builds-strategy-docker-build.adoc

```diff
@@ -6,4 +6,9 @@
 [id="builds-strategy-docker-build_{context}"]
 = Docker build
 
-The docker build strategy invokes the docker build command, and it expects a repository with a Dockerfile and all required artifacts in it to produce a runnable image.
+{product-title} uses Buildah to build a container image from a Dockerfile. For more information on building container images with Dockerfiles, see link:https://docs.docker.com/engine/reference/builder/[the Dockerfile reference documentation].
+
+[TIP]
+====
+Also see link:https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact[Understand how ARG and FROM interact] in the Dockerfile reference documentatiosn.
```

Comment:
IMO, the "top level"  link that @adambkaplan had you add make sense there; but this more precise tip needs to context of "I'm manipulating build args".

I would contend this needs to be in the https://docs.openshift.com/container-platform/4.7/cicd/builds/build-strategies.html#builds-strategy-docker-build-arguments_build-strategies section

But maybe @adambkaplan does not agree .... we'll see what he says when he gets a chance to chime in.

Reply:
I don't think the tip here is necessary. Instead we should link to the Dockerfile `ARG` reference in the build args section of our docs: https://docs.openshift.com/container-platform/4.7/cicd/builds/build-strategies.html#builds-strategy-docker-build-arguments_build-strategies

It looks like that section of the docs has a bad link anyway. The Dockerfile external link goes into the right level of detail wrt to how ARG behaves, and I would rather we don't duplicate content and risk it getting stale over time.

Reply:
OK so @adambkaplan agrees with me @rolfedh about moving this ref/tip to https://docs.openshift.com/container-platform/4.7/cicd/builds/build-strategies.html#builds-strategy-docker-build-arguments_build-strategies

Give that a go please.